introduction-to-bedrock/bedrock_mistral.py

```diff
@@ -0,0 +1,100 @@
+import boto3
+import json
+
+#Create the connection to Bedrock
+bedrock = boto3.client(
+    service_name='bedrock',
+    region_name='us-west-2', 
+)
+
+bedrock_runtime = boto3.client(
+    service_name='bedrock-runtime',
+    region_name='us-west-2', 
+)
+
+# Let's see all available Meta Models
+
+available_models = bedrock.list_foundation_models()
+for model in available_models['modelSummaries']:
+  if model['providerName']=='Mistral AI':
+    print(json.dumps(model, indent=2))
+
+# Define prompt and model parameters
+# - Mistral Instruct chat template is -- similiar to <s>[INST] Instruction [/INST] Model answer</s>[INST] Follow-up instruction [/INST]
```

Comment:
I don't think <s></s> is required for bedrock is it?

Reply:
it is good to have a sample of the prompt template -- it is recomended by Mistral